src/plugins/vis_default_editor/public/components/controls/utils/use_handlers.ts

```diff
@@ -7,6 +7,7 @@
  */
 
 import { useCallback } from 'react';
+import { SerializableRecord } from '@kbn/utility-types';
```

Comment:
nit: please add import **type** { SerializableRecord } from '@kbn/utility-types'; and try to do that in future for importing types

Reply:
done